backend/pyproject.toml

```diff
@@ -30,7 +30,7 @@ dependencies = [
     "python-socketio==5.9.0", # For log_events
     "social-auth-app-django==5.3.0", # For OAuth
     "social-auth-core==4.4.2", # For OAuth
-    "unstract-sdk~=0.20.0",
+    "unstract-sdk~=0.20.1",
```

Comment:
Patch version gets automatically picked up as long the dependency is explicitly included

Reply:
Yes @gaya3-zipstack . Just added it to track SDK version in tool.